src/Calculator/Views/GraphingCalculator/GraphingSettings.xaml.cs

```diff
@@ -110,11 +110,9 @@ private void GridSettingsTextBox_PreviewKeyDown(object sender, KeyRoutedEventArg
         {
             if (e.Key == VirtualKey.Enter)
             {
-                // CSHARP_MIGRATION: TODO:
-                // check if condition
-                if (FocusManager.TryMoveFocusAsync(FocusNavigationDirection.Next).Status != AsyncStatus.Completed)
+                if (FocusManager.TryMoveFocusAsync(FocusNavigationDirection.Next) == null)
```

Comment:
Per the original condition, should we use *asyncResult == null ||  asyncResult.Status != AsyncStatus.Completed*?

Reply:
here's the original condition in C++/CX
```
if (!FocusManager::TryMoveFocusAsync(FocusNavigationDirection::Next))
{
  FocusManager::TryMoveFocusAsync(FocusNavigationDirection::Previous);
}
```
It's weird but I don't think it was equivalent to `asyncResult.Status != AsyncStatus.Completed`